styles.css

```diff
@@ -371,44 +376,203 @@ textarea {
-    display:none;
+
+  .menu-mobile {
+    display: none;
   }
-  .menu-desktop{
+
+  .menu-desktop {
     display: block;
     color: #344563;
     font-size: 15px;
     font-weight: 600;
+    text-decoration: none;
   }
+
   .headline-container {
     background-image: none;
-}
-  .desktop-background{
+  }
+
+  .desktop-background {
     background-image: url("./images/header_desktop_img.png");
     background-repeat: no-repeat;
-    background-size: cover;
+    background-size: 100%;
     padding-top: 36px;
+    padding-bottom: 150px;
+    text-align: center;
+  }
+
+  .myinfo {
+    padding-top: 160px;
+  }
+
+  .myinfo > * {
+    font-size: 48px;
+  }
+
+  .info {
+    font-size: 20px;
+    margin-left: 260px;
+    margin-right: 260px;
+  }
+
+  .action_button {
+    display: none;
+  }
+
+  .action_button_1_container {
+    display: block;
+    margin-top: 84px;
+  }
+
+  .action_button_1 {
+    width: 40px;
+    height: 40px;
+  }
+
+  .works-line {
+    display: none;
+  }
+
+  .works {
+    grid-template-columns: auto auto auto;
+    grid-template-rows: auto auto auto;
+    grid-row-gap: 32px;
+    grid-column-gap: 24px;
+    margin-top: 100px;
+    padding-left: 120px;
+    padding-right: 120px;
+  }
+
+  .works-title {
+    grid-column: 1/4;
+    grid-row: 1/1;
+  }
+
+  .about-me {
+    grid-template-columns: auto auto auto;
+    grid-template-rows: auto;
+    grid-column-gap: 24px;
+  }
+
+  .about-me-title {
+    grid-row: 1/1;
+    grid-column: 1/4;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: url("./images/about_me_desktop_img.png");
+    background-size: 100%;
+  }
+
+  .about-me-title p {
+    margin-left: 260px;
+    margin-right: 260px;
+  }
+
+  .about-me-card:nth-child(2) {
+    margin-left: 120px;
+  }
+
+  .about-me-card:nth-child(4) {
+    margin-right: 120px;
+  }
+
+  .about-me-card ul {
+    justify-content: center;
+  }
+
+  .contact-me {
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin-top: 100px;
+    padding-left: 140px;
+    padding-right: 140px;
+    column-gap: 24px;
+    align-items: flex-start;
+    background-image: url("./images/contact_me_desktop_img.png");
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: 100%;
+  }
+
+  .email-container {
+    display: none;
+  }
+
+  .bottom-line {
+    display: none;
+  }
+
+  .contact-text-container {
+    flex-basis: 42%;
+    padding-top: 30px;
+  }
+
+  .contact-text {
+    font-size: 30px;
+    line-height: 52px;
+    margin: 0;
+  }
+
+  form div {
+    padding-left: 0;
+  }
+
+  .contact-me div {
+    margin-left: 0;
+    margin-right: 0;
+    text-align: left;
+  }
+
+  .form-container {
+    flex-grow: 1;
+  }
+
+  .contact-me .social-media {
+    flex-basis: 100%;
+    padding-top: 150px;
+  }
+
+  .menu-desktop:hover {
+    color: #008552;
+  }
+
+  button:active {
+    background-color: #008552;
+  }
+
+  button:hover {
+    cursor: pointer;
+  }
```

Comment:
The buttons should have a greenish shadow when you hover over them.

<img width="114" alt="image" src="https://user-images.githubusercontent.com/87897753/160014686-1715859f-fbb8-43b1-9399-10b95204c173.png">